dependencies.gradle.kts

```diff
@@ -61,7 +61,7 @@ project.apply {
         set("buildPlatform", getBuildPlatform())
 
         // Android
-        set("minSdkVersion", 23)
+        set("minSdkVersion", 24)
```

Comment:
Can we do this? EncryptedLogging uses `org.wordpress:utils` which has `minSdk=24`.

Reply:
We discussed it recently here: p1713856697725349-slack-C028JAG44VD

But before we do anything could you explain why do we need encrypted logging? Is it required by the Sentry wrapper?

Reply:
> We discussed it recently here: p1713856697725349-slack-C028JAG44VD. In short it should be fine but we'd have to indicate it in the changelog.

Gotcha, thanks!

> But before we do anything could you explain why do we need encrypted logging? Is it required by the Sentry wrapper?

Encrypted logging will give the team more insight about what happened just before the crash. It can be used when sending a support ticket. We encrypt, instead of sending plain logs, to enhance privacy of users and don't share logs with any 3rd party. It's not required by the Sentry wrapper though, it's a separate feature/request (pdnsEh-18Z-p2)

Reply:
Ok, got it. Then I don't see any reason not to bump min SDK. cc: @Automattic/pocket-casts-android

Reply:
Yes I agree, it's such a small number of users left on 23. Let's bump the min SDK to 24.